website/content/docs/connect/l7-traffic/configuration/service-splitter.mdx

```diff
@@ -0,0 +1,396 @@
+---
+layout: docs
+page_title: Service Splitter - Configuration Entry Reference
+description: >-
+  The service splitter configuration entry kind defines how to divide service mesh traffic between service instances. Use the reference guide to learn about `""service-splitter""` config entry parameters and how it can be used for traffic management behaviors like canary rollouts, blue green deployment, and load balancing across environments.
+---
+
+# Service Splitter Configuration Entry Reference
+
+This topic provides configuration reference information for the service splitter configuration entry. 
+
+The `service-splitter` configuration entry kind (`ServiceSplitter` on Kubernetes) splits incoming requests across different subsets of a single service (like during staged canary rollouts), or across different services (like during a v2 rewrite or other type of codebase migration).  Since L7 traffic splitting is only meant for HTTP, you cannot use request headers with a TCP listener.
+
+## ACLs
+
+Configuration entries may be protected by [ACLs](/docs/security/acl).
```

Comment:
Hello 👋🏻 Now that the[ link migration ](https://github.com/hashicorp/consul/pull/15976)has been merged, I'm running through open PRs to make notes on updating legacy links. Please let me know if you have questions. 
```suggestion
Configuration entries may be protected by [ACLs](/consul/docs/security/acl).
```